input/examples/location-murrabit-hospital.xml

```diff
@@ -1,3 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Compliant with HI Vendor Test Data -->
```

Comment:
Remove the comment Compliant with HI Vendor Test Data for Location

Reply:
Done